Jenkinsfile

```diff
@@ -0,0 +1,14 @@
+buildMvn {
+  publishModDescriptor = 'no'
+  mvnDeploy = 'yes'
+  publishAPI = 'yes'
+  runLintRamlCop = 'yes'
+
+  doDocker = {
+    buildJavaDocker {
+      publishMaster = 'yes'
+      healthChk = 'yes'
+      healthChkCmd = 'curl -sS --fail -o /dev/null  http://localhost:8081/admin/health || exit 1'
```

Comment:
If we want to have this as healthChkCmd, we might want to add a user story to implement `/admin/health` endpoint, otherwise this health check will always fail